src/reducer.js

```diff
@@ -0,0 +1,109 @@
+
+import { Rectangle, RoundedRectangle, Circle } from 'components/Shapes';
+
+const SELECT_SHAPE = 'SELECT_SHAPE';
+const LOAD_SHADOW_PROPERTY = 'LOAD_SHADOW_PROPERTY';
+const UPDATE_SHADOW_PROPERTY = 'UPDATE_SHADOW_PROPERTY';
+
+const initialState = {
+  ActiveShape: Rectangle,
+  shapes: [
+    {
+      id: 'rectangle',
+      Shape: Rectangle,
+      isSelected: true
+    },
+    {
+      id: 'rounded-rectangle',
+      Shape: RoundedRectangle,
+      isSelected: false
+    },
+    {
+      id: 'circle',
+      Shape: Circle,
+      isSelected: false
+    }
+  ],
```

Comment:
1. A component should never be stored in state. Reference: https://github.com/reduxjs/redux/issues/1248
2. Also, I don't think there is a need for the key `ActiveShape`. ActiveShape is just an element in `shapes` array where `isSelected` is true, isn't it? So either you have an `activeShape` or you have a `shapes` array, but not both.